docker-ee-for-windows/install.md

```diff
@@ -0,0 +1,152 @@
+    PS> Install-Package -Name docker -ProviderName DockerMsftProvider -Force
+    PS> Restart-Computer -Force
+    ```
+
+2.  Test your Docker EE installation by running the `hello-world` container.
+
+    ```ps
+    PS> docker run hello-world:nanoserver
+
+    Unable to find image 'hello-world:nanoserver' locally
+    nanoserver: Pulling from library/hello-world
+    bce2fbc256ea: Pull complete
+    3ac17e2e6106: Pull complete
+    8cac44e17f16: Pull complete
+    5e160e4d8db3: Pull complete
+    Digest: sha256:25eac12ba40f7591969085ab3fb9772e8a4307553c14ea72d0e6f98b2c8ced9d
+    Status: Downloaded newer image for hello-world:nanoserver
+
+    Hello from Docker!
+    This message shows that your installation appears to be working correctly.
+    <snip>
+    ```
+
+
+## Using a script to install Docker EE
+
+Use the following steps when you want to install manually, script automated
+installs, or install on air-gapped systems.
+
+1.  In a PowerShell command prompt, download the installer archive on a machine
+that has a connection.
+
+    ```ps
+    # On an online machine, download the zip file.
+    PS> invoke-webrequest -UseBasicparsing -Outfile docker.zip https://download.docker.com/components/engine/windows-server/17.03/docker-17.03.0-ee.zip
+    ```
+
+2.  Copy the zip file to the machine where you want to install Docker. In a
+PowerShell command prompt, use the following commands to extract the archive,
+register, and start the Docker service.
+
+    ```ps
+    # Extract the archive.
+    PS> Expand-Archive docker.zip -DestinationPath $Env:ProgramFiles
+
+    # Clean up the zip file.
+    PS> Remove-Item -Force docker.zip
+
+    # Install Docker. This will require rebooting.
+    $null = Install-WindowsFeature containers
+
+    # Add Docker to the path for the current session.
+    PS> $env:path += "$env:ProgramFiles\docker"
+
+    # Optionally, modify PATH to persist across sessions.
+    PS> $newPath = "$env:ProgramFiles\docker;" +
+    [Environment]::GetEnvironmentVariable("PATH",
+    [EnvironmentVariableTarget]::Machine)
+
+    PS> [Environment]::SetEnvironmentVariable("PATH", $newPath,
+    [EnvironmentVariableTarget]::Machine)
+
+    # Register the Docker daemon as a service.
+    PS> dockerd --register-service
+
+    # Start the daemon.
+    PS> Start-Service docker
+    ```
+
+3.  Test your Docker EE installation by running the `hello-world` container.
+
+    ```ps
+    PS> docker run hello-world:nanoserver
+    ```
+
+
+## Install Docker EE using OneGet
+
+If you want to install Docker EE by using [OneGet](https://github.com/oneget/oneget),
+follow the steps described in [Windows Containers on Windows
+Server](https://docs.microsoft.com/en-us/virtualization/windowscontainers/quick-start/quick-start-windows-server).
+
+##  What to know before you install
+
+* **What the Docker EE for Windows install includes**: The installation
+provides [Docker Engine](/engine/userguide/intro.md) and the
```

Comment:
Same nit here with the indentation. The 'provides' should line up with the first asterisk before 'What'.